scripts/launch-xrd

```diff
@@ -413,7 +420,7 @@ else
         # know the major device num to pass as e.g.
         # --device-cgroup-rule 'c 244:* rwm' (where the number can vary on
         # different hosts).
-        if [[ $CONTAINER_MGR == docker ]]; then
+        if [[ $CONTAINER_EXE =~ "docker" ]]; then
```

Comment:
CONTAINER_MGR wasnt being used anywhere. So I removed it. Should I keep it? As Paulo mentioned, it can be used in the `_get_platforms` error message ? Is it worth keeping for just an error message =?

Reply:
It was being used in places like this, arguably it was clearer before (rather than needing to remember to use regex matching every time). And yeah there's the error message too - I'd keep `CONTAINER_MGR`.